components/ide-proxy/Dockerfile

```diff
@@ -18,9 +18,11 @@ RUN for FILE in `ls /bin/gitpod-local-companion*`;do \
   gzip -v -f -9 -k "$FILE"; \
 done
 
+RUN mkdir -p static/code
+RUN wget https://raw.githubusercontent.com/gitpod-io/openvsx.org.json/main/marketplace.json -O static/code/marketplace.json
```

Comment:
If you use main branch directly, it doesn't do what you want it to do, and it even causes some other unexpected problems because you don't know under what circumstances it will trigger the build, which means we have no idea which version of the compiled image contains the file, I suggest using commitid here

Reply:
@mustard-mh I understand that this may be problematic, but what exactly could these problems lead to? I would like to keep it on `main` since we always want the latest version, which wouldn't require a commit to this repository.

Reply:
Let's go with this now 👍